text/lang-detection/Project.toml

```diff
@@ -4,10 +4,4 @@ Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Gumbo = "708ec375-b3d6-5a57-a7ce-8257bf98657a"
 HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
-
-[compat]
-Flux = "0.11.4"
-julia = "1.5"
```

Comment:
These deps and compat entries should be kept. The latter just need a bump, and if you wouldn't mind I think adding compat entries for the other packages would be good too.

Reply:
That said, removing Parameters in favour of `Base.@kwdef` could save one dependency if you're up for it.

Reply:
Thank you. I have changed the parameters to `Base.@kwdef` and updated `Project.toml`. For the compat entries, I tested with julia 1.6.6; other packages than Flux are just the latest.